best-practices/go-ci.md

```diff
@@ -27,6 +27,23 @@ should also run on 1.15.x. Note that upstream Go supports
 [two major releases](https://github.com/golang/go/wiki/Go-Release-Cycle#release-maintenance)
 at once as well.
 
+### Using Code Coverage Tools
+
+Code coverage should be determined in CI, but should not be used as an automatic status checks.
```

Comment:
`s/should be/can be/`, maybe? I don't think we necessarily want to encourage people to use CI for it. one could collect the same data locally.

Reply:
Makes sense.